src-a1/main/java/com/example/a1/controller/ConsoleController.java

```diff
@@ -0,0 +1,242 @@
+package com.example.a1.controller;
+
+import com.example.a1.entity.Tag;
+import com.example.a1.entity.User;
+import com.example.a1.exception.NotAllowedException;
+import com.example.a1.service.AnswerManagementService;
+import com.example.a1.service.QuestionManagementService;
+import com.example.a1.service.UserManagementService;
+import com.example.a1.service.VoteManagementService;
+import jdk.nashorn.internal.objects.Global;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+@Component
+@RequiredArgsConstructor
+@Data
+public class ConsoleController implements CommandLineRunner {
```

Comment:
You should remove this class; it does not make sense to also allow the user to use the command line interface for a web app.